### PR TITLE
Send new event to snapshot-discord

### PR DESCRIPTION
**Pull request by ChaituVR**
Related to https://github.com/snapshot-labs/snapshot-hub/issues/273#issuecomment-1066145189 step 3

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -16,3 +16,4 @@ SERVICE_PUSHER_BEAMS_INSTANCE_ID=
 SERVICE_PUSHER_BEAMS_SECRET_KEY=
 SNAPSHOT_URI=http://localhost:3000
 DEFAULT_NETWORK=1
+SNAPSHOT_WEBHOOK_URL=https://snapshot-discord.herokuapp.com
```
Comment (bonustrack) [inline .env.example:19]: This should be empty by default, and if not exist we shouldnt send any event, (for testnet for example we dont need that)

File: src/events/index.ts [hunk 2]
```diff
@@ -13,6 +13,8 @@ const serviceEventsSalt = parseInt(process.env.SERVICE_EVENTS_SALT || '12345');
 const servicePushNotifications = parseInt(
   process.env.SERVICE_PUSH_NOTIFICATIONS || '0'
 );
+const webHookURL =
```
Comment (bonustrack) [inline src/events/index.ts:16]: Its webhook not web hook
```suggestion
const webhookURL =
```

File: src/events/index.ts [hunk 3]
```diff
@@ -61,6 +63,29 @@ async function sendEvent(event, to) {
   return res.json();
 }
 
+export async function sendEventToWebhook(event) {
+  // TODO: handle errors and retry, maybe save in temp table
+  try {
+    // TODO: This secret is temporary key, replace this with event URL once events code is removed
+    const secret = sha256(`${webHookURL + '/api/webhook'}${serviceEventsSalt}`);
```
[the remark this answers is not in the document]
Comment (ChaituVR) [reply src/events/index.ts:70]: Hey yes, this is to reuse the secret token of webhook here 😅 i added a comment on the above line

```JS
    // TODO: This secret is a temporary key, replace this with the event URL once the events code is removed
```
We can change it in a later step :)

**Comment by ChaituVR** [comment]
@bonustrack Changed according to comments, please review again whenever you get some time

File: src/events.ts [hunk 4]
```diff
@@ -65,6 +66,30 @@ async function sendEvent(event, to) {
   return res.json();
 }
 
+export async function sendEventToWebhook(event) {
+  if (!webhookURL) return;
+  // TODO: handle errors and retry, maybe save in temporary table
+  try {
+    // TODO: This secret is temporary key, replace this with event URL once events code is removed
+    const secret = sha256(`${webhookURL}/api/webhook'${serviceEventsSalt}`);
```
Comment (ChaituVR) [inline src/events.ts:74]:
```suggestion
    const secret = sha256(`${webhookURL}${process.env.SERVICE_EVENTS_SALT}`)
```

Comment (ChaituVR) [inline src/events.ts:74]:
```suggestion
    const secret = sha256(`${webhookURL}/api/webhook?cb=1'${serviceEventsSalt}`);
```

File: .env.example [hunk 5]
```diff
@@ -12,3 +12,4 @@ SERVICE_PUSHER_BEAMS_INSTANCE_ID=
 SERVICE_PUSHER_BEAMS_SECRET_KEY=
 SNAPSHOT_URI=http://localhost:3000
 DEFAULT_NETWORK=1
+SNAPSHOT_WEBHOOK_URL=
```
Comment (ChaituVR) [inline .env.example:15]:
```suggestion
SNAPSHOT_WEBHOOK_URL=https://webhook.snapshot.org
```

File: src/events.ts [hunk 6]
```diff
@@ -65,6 +66,30 @@ async function sendEvent(event, to) {
   return res.json();
 }
 
+export async function sendEventToWebhook(event) {
+  if (!webhookURL) return;
+  // TODO: handle errors and retry, maybe save in temporary table
+  try {
+    // TODO: This secret is temporary key, replace this with event URL once events code is removed
+    const secret = sha256(`${webhookURL}/api/webhook?cb=1'${serviceEventsSalt}`);
```
Comment (ChaituVR) [inline src/events.ts:74]:
```suggestion
    const secret = sha256(`${webhookURL}/api/webhook?cb=1${serviceEventsSalt}`);
```

File: src/events.ts [hunk 7]
```diff
@@ -65,6 +66,30 @@ async function sendEvent(event, to) {
   return res.json();
 }
 
+export async function sendEventToWebhook(event) {
+  if (!webhookURL) return;
+  // TODO: handle errors and retry, maybe save in temporary table
+  try {
+    // TODO: This secret is temporary key, replace this with event URL once events code is removed
+    const secret = sha256(`${webhookURL}/api/webhook?cb=1${serviceEventsSalt}`);
+    const res = await fetch(webhookURL + '/api/event', {
```
Comment (bonustrack) [inline src/events.ts:75]: We should always use template literals
```suggestion
    const res = await fetch(`${webhookURL}/api/event`, {
```

**Pull request merged**